src/components/textbox/textbox.component.tsx

```diff
@@ -358,7 +379,9 @@ export const Textbox = React.forwardRef(
           >
             {characterLimit || inputHint ? (
               <StyledInputHint id={hintIdValue} data-element="input-hint">
-                {characterCountHint || inputHint}
+                {formattedCharacterCountHintString ||
+                  characterCountHintString ||
+                  inputHint}
```

Comment:
**suggestion**: likewise you might be able to use @edleeks87's suggestion for `Textbox` too :)